exercises/resistor-color-duo/resistor-color-duo.ts

```
@@ -0,0 +1,6 @@
+
+
+
+export const value = (_: string[]): number => {
```

Are they other ways to solve this?

---

Please throw an error.

---

Wrapped the value method in a class and updated example code to throw error and is covered via test.